plugins/must-gather/skills/must-gather-analyzer/scripts/analyze_events.py

```
@@ -0,0 +1,201 @@
+        else:
+            return "<1m"
+    except Exception:
+        return ""
+
+
+def format_event(event: Dict[str, Any]) -> Dict[str, Any]:
+    """Format an event for display."""
+    metadata = event.get('metadata', {})
+
+    namespace = metadata.get('namespace', '')
+    name = metadata.get('name', 'unknown')
```

Ruff (0.14.4)

Local variable `name` is assigned to but never used

Remove assignment to unused variable `name`

(F841)